frontend/src/app/components/users/birthday/users.birthday.controller.js

```diff
@@ -44,19 +44,19 @@ export default class BirthdayController {
             if (Array.isArray(response.data) && response.data.length > 0) {
                 this.employees = response.data;
                 let index = this.employees.length - 1;
-                this.employee_name = this.employees[index].firstName + ' ' + this.employees[index].lastName;
+                this.employeeName = this.employees[index].firstName + ' ' + this.employees[index].lastName;
                 this.employeeImg = this.employees[index].image;
             } else {
                 if (Object.keys(response.data).length > 0) {
                     this.employee = response.data;
-                    this.employee_name = this.employee.firstName + ' ' + this.employee.lastName;
+                    this.employeeName = this.employee.firstName + ' ' + this.employee.lastName;
                     this.employeeImg = this.employee.image;
                 } else {
                     this.employeeImg = this.defaultImg;
                 }
             }
         }, (error) => {
-            console.log(error, 'can not get data.');
+            console.er(error, 'can not get data.');
```

Comment:
`console.er` does not exist, this will raise an error

Reply:
fixed